src/upload/form/mod.rs

```diff
@@ -117,8 +117,14 @@ pub fn build(
                         let entry = entry?;
                         let path = entry.path();
                         if path.is_file() {
-                            log::info!("Adding module {}", path.display());
-                            modules.push(Module::new(path.to_owned())?);
+                            let name = path
+                                .strip_prefix(config.upload_dir.clone())?
+                                .to_owned()
+                                .as_path()
+                                .display()
```

Comment:
This should use the `to_str` method of path and return an error if the conversion fails (due to non-UTF-8 in the path name)

Reply:
presumably for a path like like

`/home/matt/worker/dist/nested/nested/index.js`

This returns `nested/nested/index.js`?